.github/workflows/cd.yml

```diff
@@ -0,0 +1,32 @@
+name: cd
+on:
+  workflow_run:
+    workflows: ["ci"]
+    branches-ignore: [main]
+    types:
+    - completed
+  push:
+    tags:
+    - "v[0-9]+.[0-9]+.[0-9]+"
```

Comment:
You will have duplicate events because CD will run on tag push **AND** when the CI finishes on tag push

Reply:
This is based on the [mattermost-cloud](https://github.com/mattermost/mattermost-cloud) pipelines, that were made by @spirosoik. I'm unsure if this is intentional or we have bad triggers on other repos as well.

Reply:
actually doesn't run if I remember correctly if the status is not completed to the other one. It's hard dependency when I tested this.